doc/latex.rst

```diff
@@ -65,433 +49,487 @@ avoid interpretation as escape sequences, or use raw strings (as is done here).
-     .. versionchanged:: 1.6
-        ``'lualatex'`` uses same default setting as ``'xelatex'``
-     .. versionchanged:: 1.7.6
-        For French, ``xelatex`` and ``lualatex`` default to using
-        ``babel``, not ``polyglossia``.
-
-  ``'fontpkg'``
-     Font package inclusion, the default is ``'\\usepackage{times}'`` which
-     uses Times for text, Helvetica for sans serif and Courier for monospace.
-
-     .. versionchanged:: 1.2
-        Defaults to ``''`` when the :confval:`language` uses the Cyrillic
-        script.
-     .. versionchanged:: 2.0
-        Support for individual Greek and Cyrillic letters:
-
-        - In order to support occasional Cyrillic (физика частиц)
-          or Greek letters (Σωματιδιακή φυσική) in
-          a document whose language is English or a  Latin European
-          one, the default set-up is enhanced (only for ``'pdflatex'``
-          engine) to do:
+Keys that you may want to override include:
+
+``'papersize'``
+   Paper size option of the document class (``'a4paper'`` or
+   ``'letterpaper'``)
+
+   :Default: ``'letterpaper'``
+
+``'pointsize'``
+   Point size option of the document class (``'10pt'``, ``'11pt'`` or
+   ``'12pt'``)
+
+   :Default: ``'10pt'``
+
+``'pxunit'``
+   The value of the ``px`` when used in image attributes ``width`` and
+   ``height``. The default value is ``'0.75bp'`` which achieves
+   ``96px=1in`` (in TeX ``1in = 72bp = 72.27pt``.) To obtain for
+   example ``100px=1in`` use ``'0.01in'`` or ``'0.7227pt'`` (the latter
+   leads to TeX computing a more precise value, due to the smaller unit
+   used in the specification); for ``72px=1in``, simply use ``'1bp'``; for
+   ``90px=1in``, use ``'0.8bp'`` or ``'0.803pt'``.
+
+   :Default: ``'0.75bp'``
+
+   .. versionadded:: 1.5
+
+``'passoptionstopackages'``
+   A string which will be positioned early in the preamble, designed to
+   contain ``\\PassOptionsToPackage{options}{foo}`` commands.
+
+   :Default: ``''``
+
+   .. versionadded:: 1.4
+
+``'babel'``
+   "babel" package inclusion, default ``'\\usepackage{babel}'`` (the
+   suitable document language string is passed as class option, and
+   ``english`` is used if no language.) For Japanese documents, the
+   default is the empty string.
+
+   With XeLaTeX and LuaLaTeX, Sphinx configures the LaTeX document to use
+   `polyglossia`_, but one should be aware that current `babel`_ has
+   improved its support for Unicode engines in recent years and for some
+   languages it may make sense to prefer ``babel`` over ``polyglossia``.
+
+   .. _`polyglossia`: https://ctan.org/pkg/polyglossia
+   .. _`babel`: https://ctan.org/pkg/babel
+
+   .. hint::
+
+      After modifiying a core LaTeX key like this one, clean up the LaTeX
+      build repertory before next PDF build, else left-over auxiliary
+      files are likely to break the build.
+
+    :Default:  ``'\\usepackage{babel}'`` (``''`` for Japenese documents)
```

Comment:
There is too much indentation here, the "Default" ends up in the "hint" box and the hint itself becomes a blockquote.

Reply:
Good catch. Done